Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 requests = "*"
 
 [dev-packages]
+flake8 = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3.6"
```

Comment:
As there is no way to specify a range of python versions in pipenv, it's better to leave this or remove the line completely